src/broker/Makefile.am

```diff
@@ -30,9 +30,18 @@ flux_broker_SOURCES = \
 	shutdown.h \
 	shutdown.c
 
+cmbd_LDFLAGS = -export-dynamic
```

Comment:
Automake is complaining about this line:
```
src/broker/Makefile.am:33: variable `cmbd_LDFLAGS' is defined but no program or
src/broker/Makefile.am:33: library has `cmbd' as canonical name (possible typo)
```
Should this be under broker_LDFLAGS (or something similar)?